src/app/components/Input.jsx

```diff
@@ -136,20 +137,16 @@ export default class Input extends Component {
         return (
             <div className={formClasses}>
                 {!this.props.inline && !this.props.reverseLabelOrder && this.renderLabel()}
-                {this.props.error ? (
+                {this.props.error && (
                     <div id={`input-error-${this.props.id}`} className="error-msg" role="alert">
```

Comment:
So this statement will be skipped if false?

Reply:
yes, exactly, it is conditional rendering in react